### PR TITLE
Add aad b2c it execution in china cloud

### PR DESCRIPTION
**Pull request by moarychan**
- Add aad b2c it execution in china cloud
- Remove unused method in AADAuthenticationProperties

### CODE REVIEW

File: sdk/spring/azure-spring-boot-test-aad-b2c/src/test/java/com/azure/test/aad/b2c/selenium/AADB2CSeleniumITHelper.java [hunk 1]
```diff
@@ -39,20 +42,35 @@ public AADB2CSeleniumITHelper(Class<?> appClass, Map<String, String> properties)
         super(appClass, properties);
         userEmail = AAD_B2C_USER_EMAIL;
         userPassword = AAD_B2C_USER_PASSWORD;
+        isAzureCloudGlobal = Constant.AZURE_CLOUD_TYPE_GLOBAL.equalsIgnoreCase(AZURE_CLOUD_TYPE) ? true : false;
```
Comment (backwind1233) [inline sdk/spring/azure-spring-boot-test-aad-b2c/src/test/java/com/azure/test/aad/b2c/selenium/AADB2CSeleniumITHelper.java:45]: I think `Constant.AZURE_CLOUD_TYPE_GLOBAL.equalsIgnoreCase(AZURE_CLOUD_TYPE) `  is enough

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**